agent_deploy/kubernetes/sysdig-daemonset.yaml

```diff
@@ -33,15 +33,18 @@ spec:
           path: /usr
       hostNetwork: true
       hostPID: true
- #    serviceAccount: sysdigcloud                          #OPTIONAL - OpenShift service account for OpenShift
+ #    serviceAccount: sysdigcloud                           #OPTIONAL - OpenShift service account for OpenShift
       containers:
       - name: sysdig-agent
         image: sysdig/agent
+#        imagePullPolicy: Always                            #OPTIONAL - pull up latest image every deployment 
         securityContext:
          privileged: true
         env:
         - name: ACCESS_KEY                                  #REQUIRED - replace with your Sysdig Cloud access key
           value: 8312341g-5678-abcd-4a2b2c-33bcsd655
+#        - name: RUN_MODE                                   #OPTIONAL - driverless mode (limited functionality. No kernel module)
```

Comment:
In the comment I think it is better also to indicate the agent version from when this option is available